packages/KSkeleton/FullScreenKongSkeleton.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div
+    class="fullscreen-loading-container"
+    data-testid="full-screen-loader">
```

Comment:
i wonder if this needs to be in here. this is Kongponents and `data-testid` is for Kong specific testing right?

Reply:
data-testid is actually just a testing strategy so that implementation (normal dom attributes) can change, but you have selectors explicitly for testing. Eventually we'd be able to use sugar methods provided by the testing library: https://github.com/vuejs/vue-test-utils/issues/1500

Reply:
ok, this is the first time we've used it in Kongponents so wanted to check